src/runtime_src/core/edge/skd/xrt_skd.cpp

```diff
@@ -44,8 +44,8 @@ namespace xrt {
     memcpy(xclbin_uuid,uuid,sizeof(xclbin_uuid));
 
     // Set sk_path according to sk_name
-    snprintf(sk_path, XRT_MAX_PATH_LENGTH, "%s%s%d", SOFT_KERNEL_FILE_PATH,
-	     sk_name,cu_idx);
+    snprintf(sk_path, XRT_MAX_PATH_LENGTH, "%s%s", SOFT_KERNEL_FILE_PATH,
+	     sk_name);
   }
```

Comment:
Can we use std::string and friends here instead of snprintf?

Reply:
That change is in another PR 7032.